src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
@LaurentGoderre Whoops - looks like I totally botched my commit's message (any by extension - this PR's title). It was stating the opposite of what I intended. That's probably what caused (or at least contributed to) all this confusion. Sorry :(.

This is what I should've called it (will change it in a sec):
> Menu: In noscript, only **~~hide~~ show** menus/footer in screen viewports.

My commit's SCSS changes are what I intended. My goal was to use a noscript, screen-only placeholder selector. I researched/tested it beforehand. It also looks ok in compiled noscript.css files and sasslint is happy with it.

@duboisp I wanted to make ``%noscript-display-block-important`` available in the global noscript file since it's generic-enough to potentially be used by more than just the menu plugin. Even if it starts out with only one plugin using it, there's nothing stopping other plugins from potentially adopting it in the future.

I might rework this PR soon to make the Lightbox and Overlay plugins use ``%noscript-display-block-important``.